torchvision/datasets/kinetics.py

```diff
@@ -247,6 +248,9 @@ class Kinetics400(Kinetics):
     `Kinetics-400 <https://deepmind.com/research/open-source/open-source-datasets/kinetics/>`_
     dataset.
 
+    .. deprecated:: 0.12.0
+        Please use ``Kinetics(..., num_classes='400')`` instead. Will be removed in ``0.14.0``.
+
```

Comment:
I don't know why but unfortunately this isn't rendered properly by our sphinx team. I guess we'll have to switch to warnings for it to be visible. We'll also have to indicate the 0.12 version in the message

```suggestion
    .. warning::
        This class was deprecated in 0.12 and will be removed in 0.14. Please use ``Kinetics(..., num_classes='400')`` instead.

```